0001-kaa-protocol/README.md

```diff
@@ -1,9 +1,9 @@
 ---
-name: Kaa Protocol
+name: Kaa protocol
 shortname: 1/KP
 status: draft
-editor: Alexey Shmalko <ashmalko@cybervisiontech.com>
-contributors: Alexey Gamov <agamov@cybervisiontech.com>, Andrew Kokhanovskyi <akokhanovskyi@cybervisiontech.com>
+editor: Alexey Shmalko<ashmalko@cybervisiontech.com>
+contributors: Alexey Gamov<agamov@cybervisiontech.com>, Andrew Kokhanovskyi<akokhanovskyi@cybervisiontech.com>
```

Comment:
Why? With spaces, it results in redundant space before comma: "Alexey Gamov , Andrew Kokhanovskyi"